CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Turbolinks (master)
 
+*   Prevent `Turbolinks.visit` to the same-as-current cached page with `change` option from scrolling up.
```

Comment:
No need for a changelog entry as this is changing a feature (`change` option) that hasn't been released yet.